src/screen/splash.rs

```diff
@@ -42,9 +41,7 @@ fn spawn_splash(mut commands: Commands, asset_server: Res<AssetServer>) {
                         width: Val::Percent(70.0),
                         ..default()
                     },
-                    image: UiImage::new(
-                        asset_server.load("embedded://bevy_template/screen/splash/splash.png"),
-                    ),
+                    image: UiImage::new(asset_server.load("splash.png")),
```

Comment:
There's no way to reconcile this being an embedded asset with trunk?

Reply:
[`e5f642c` (#95)](https://github.com/TheBevyFlock/bevy-template/pull/95/commits/e5f642cd0727910094b6c4559c963ea604546d6c)